### PR TITLE
Added docker test

### PR DESCRIPTION
**Pull request by omusil24**

### CODE REVIEW

File: test/docker_test/test.sh [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+export MAKE_JOBS=4
+
+# build module and install
+echo && echo "-- building module --"
+cd ${MODULE_SRC_DIR}
+mkdir build
+cd build
+cmake .. -DCMAKE_BUILD_TYPE=debug -DCMAKE_INSTALL_PREFIX=${INSTALL_PREFIX}
+make -j${MAKE_JOBS}
+make install
```
Comment (turlyalz) [inline test/docker_test/test.sh:35]: Do we have tests to run?

File: test/docker_test/test.sh [hunk 2]
```diff
@@ -0,0 +1,35 @@
...
+
+echo "export QORE_UID=999" >> ${ENV_FILE}
+echo "export QORE_GID=999" >> ${ENV_FILE}
+
+. ${ENV_FILE}
+
+export MAKE_JOBS=4
+
+# build module and install
+echo && echo "-- building module --"
```
Comment (turlyalz) [inline test/docker_test/test.sh:29]: building module-sysconf?

File: test/docker_test/test.sh [hunk 3]
```diff
@@ -0,0 +1,35 @@
...
+cd ${MODULE_SRC_DIR}
+mkdir build
+cd build
+cmake .. -DCMAKE_BUILD_TYPE=debug -DCMAKE_INSTALL_PREFIX=${INSTALL_PREFIX}
```
Comment (turlyalz) [inline test/docker_test/test.sh:33]: We don't need to set the build type, right? I suppose that it's only for tests in gitlab?

File: test/docker_test/test.sh (hunk 1 above)
Comment (omusil24) [reply test/docker_test/test.sh:35]: No tests here.

File: test/docker_test/test.sh (hunk 3 above)
Comment (omusil24) [reply test/docker_test/test.sh:33]: No... It's just copied code from the other module builds.

**Pull request merged**